webapp/channels/src/components/claim/components/email_to_ldap.tsx

```diff
@@ -233,4 +234,4 @@ const EmailToLDAP = ({email, siteName, ldapLoginFieldName}: Props) => {
     );
 };
 
-export default EmailToLDAP;
+export default injectIntl(EmailToLDAP);
```

Comment:
Since this is a functional component you can use the hook `useIntl` directly instead of using this hoc 
Ref https://formatjs.io/docs/react-intl/api/#useintl-hook

There are many places in code that we use this hook already :)

Reply:
Sounds good!
I used it become it was mentioned in the issue description and the example PR. Might be useful to add a line about usage of this hook in the issue descriptions

Reply:
i did mention it

> If the component uses translated strings as titles or placeholders, use the correct React-Intl API, such as the 'formattedMessage' hook or the 'injectIntl' higher-order component.

but i think it can be improved more

Reply:
Applied